modules/complete-eks-cluster/main.tf

```diff
@@ -33,6 +33,7 @@ module "eks-cluster" {
   cluster_enabled_log_types      = var.cluster_enabled_log_types
   cluster_version                = var.cluster_version
   map_roles                      = var.map_roles
+  create_cluster                 = var.create_cluster
```

Comment:
do we use this option?

Reply:
yes, for example there is a case we don't need to create a cluster but have to import existing resources